holiday-calendar-base/src/main/lombok/com/github/davejoyce/calendar/FixedHoliday.java

```diff
@@ -39,6 +41,8 @@
  */
 public class FixedHoliday extends Holiday {
 
+    @Getter
+    @NonNull
```

Comment:
Reading the docs for `@NonNull` makes me wonder whether this does anything on a field given that Lombok is not generating any of the constructors.  I think you would need to annotate the parameter on the constructor that takes this value for it to take effect?